cephfs/admin/clone_progress_test.go

```diff
@@ -0,0 +1,73 @@
+//go:build ceph_main
```

Comment:
```suggestion
//go:build main
```
I know it works the other way also, but for the sake of completeness, the approach mostly followed in go-ceph is to just use the release version/branch name in build constraints.